README.md

```diff
@@ -93,6 +93,7 @@ cd code
 git clone https://github.com/decred/decrediton.git
 git clone https://github.com/grpc/grpc
 cd grpc
+git checkout cc2e048e84eaa418cab393553594a3fefb891037  # linux, on OSX use commit 42a95a21e31e3c604e195a9d3ac50af4941436bb instead
```

Comment:
`cc2e048e84eaa418cab393553594a3fefb891037` worked for me, I ran into the `cares` dependency issues on `42a95a21e31e3c604e195a9d3ac50af4941436bb`